modules/aws_ecs/variables.tf

```diff
@@ -15,11 +15,17 @@ variable "vpc_id" {
   description = "Select a VPC that allows instances access to the Internet."
 }
 
-variable "subnet_ids" {
+variable "private_subnet_ids" {
```

Comment:
would replacing this var with two vars with different ids break current usage? fwiw i don't know how people use this repo right now, i imagine if they choose to copy over the tf to their own repos then it doesnt matter if we're changing things, but if they use the whole thing as a tf module it would break on upgrades?

Reply:
they would just have to add the new variable, and they can just put the same subnets in both public and private variables to mimic current usage.